susemanager-utils/susemanager-sls/src/modules/kiwi_source.py

```diff
@@ -100,7 +100,7 @@ def _prepareGit(source, dest, root):
   if rev == '':
     rev = 'master'
 
-  log.debug('GIT URL: {}, Revision: {}, subdir: {}'.format(url, rev, subdir))
+  log.debug('GIT URL: {0}, Revision: {1}, subdir: {2}'.format(url, rev, subdir))
```

Comment:
Specifying placeholders index is mandatory for Python 2.6. Is this intended?

Reply:
I guess this is because of SLE11.

Reply:
Yes, this is needed because SLES11 has python 2.6